packages/cardano-services/test/entrypoints.test.ts

```diff
@@ -863,5 +865,137 @@ describe('entrypoints', () => {
+
+        beforeEach(async () => {
+          spy = jest.fn();
+        });
+
+        it('cli:start-server exits with code 1', async (done) => {
+          proc = fork(
+            exePath('cli'),
+            [
+              'start-server',
+              '--api-url',
+              apiUrl,
+              '--postgres-connection-string',
+              postgresConnectionString,
+              '--logger-min-severity',
+              'error',
+              '--cardano-node-config-path',
+              cardanoNodeConfigPath,
+              ServiceNames.NetworkInfo,
+              '--postgres-ssl-ca-file',
+              invalidFilePath
+            ],
+            { stdio: 'pipe' }
+          );
+          proc.stderr!.on('data', (data) => {
+            spy();
+            expect(data.toString().includes('ENOENT: no such file or directory')).toEqual(true);
+          });
+
+          proc.on('exit', (code) => {
+            expect(code).toBe(1);
+            expect(spy).toHaveBeenCalled();
+            done();
+          });
+        });
+        it('run exits with code 1', (done) => {
+          expect.assertions(2);
+          ogmiosServer.listen(ogmiosConnection.port, () => {
+            proc = fork(exePath('run'), {
+              env: {
+                API_URL: apiUrl,
+                CARDANO_NODE_CONFIG_PATH: cardanoNodeConfigPath,
+                LOGGER_MIN_SEVERITY: 'error',
+                POSTGRES_CONNECTION_STRING: postgresConnectionString,
+                POSTGRES_SSL_CA_FILE: invalidFilePath,
+                SERVICE_NAMES: ServiceNames.NetworkInfo
+              },
+              stdio: 'pipe'
+            });
+            proc.stderr!.on('data', (data) => {
+              spy();
+              expect(data.toString().includes('ENOENT: no such file or directory')).toEqual(true);
+            });
+            proc.on('exit', (code) => {
+              expect(code).toBe(1);
+              expect(spy).toHaveBeenCalled();
+              done();
+            });
+          });
+        });
+      });
+
+      describe('specifying ssl ca file path', () => {
+        let spy: jest.Mock;
+
+        beforeEach(async () => {
+          spy = jest.fn();
+        });
+        it('cli:start-server exits with code 1', async (done) => {
+          proc = fork(
+            exePath('cli'),
+            [
+              'start-server',
+              '--api-url',
+              apiUrl,
+              '--postgres-connection-string',
+              postgresConnectionString,
+              '--logger-min-severity',
+              'error',
+              '--cardano-node-config-path',
+              cardanoNodeConfigPath,
+              ServiceNames.NetworkInfo,
+              '--postgres-ssl-ca-file',
+              postgresSslCaFile
+            ],
+            { stdio: 'pipe' }
+          );
+          proc.stderr!.on('data', (data) => {
+            spy();
+            expect(data.toString().includes('self signed certificate')).toEqual(true);
```

Comment:
Where is this text coming from?

Reply:
The error thrown is due to an invalid certificate `Error: DEPTH_ZERO_SELF_SIGNED_CERT: self signed certificate`, a comment claryfing this can be added. 
If we want to avoid this error I've seen a workaround by setting `process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"` but I'm not convinced with that.

Reply:
OK. I think this is fine given it demonstrates the content of the file is being loaded and passed to the underlying library.

Reply:
Updated to `The server does not support SSL connections`